smoke-test/tests/cypress/cypress/support/commands.js

```diff
@@ -95,11 +100,11 @@ Cypress.Commands.add("goToStarSearchList", () => {
 })
 
 Cypress.Commands.add("openThreeDotDropdown", () => {
-  cy.get('[data-testid="entity-header-dropdown"]').click();
+  cy.clickOptionWithTestId("entity-header-dropdown")
 });
 
 Cypress.Commands.add("clickOptionWithText", (text) => {
-  cy.contains(text).click();
+  cy.contains(text).click({force: true});
```

Comment:
why do we need this force?

Reply:
I have enabled force in some cases, and it started causing failures in the biz glossary test. I recommend looking at this closely..

Reply:
You can see `glossary.js` is failing

Reply:
fixed this. Thanks for pointing this out. Seems I was wrong here.

Reply:
All tests are passing now.